frontend/public/kinds.ts

```diff
@@ -50,7 +50,7 @@ export const connectToPlural: ConnectToPlural = connect(
       match: match<{ plural: GroupVersionKind | string }>;
     },
   ) => {
-    const plural = props.plural || _.get(props, 'match.params.plural');
+    const plural = props.plural ?? props.match?.params?.plural;
```

Comment:
I'm not sure if this could be an issue, but this does change the behavior if `props.plural` is the empty string.